develop/dev-guide-third-party-support.md

```diff
@@ -257,18 +257,18 @@ If you encounter problems when connecting to TiDB using the tools listed in this
          <td>N/A</td>
       </tr>
       <tr>
-         <td><a href="https://github.com/coleifer/peewee/" target="_blank" referrerpolicy="no-referrer-when-downgrade">peewee</a></td>
-         <td>v3.14.10</td>
-         <td>Compatible</td>
+         <td><a href="https://www.sqlalchemy.org/" target="_blank" referrerpolicy="no-referrer-when-downgrade">SQLAlchemy</a></td>
```

Comment:
Please update the support level of SQLAlchemy in dev-guide-choose-driver-or-orm.md:

https://github.com/pingcap/docs/blob/b6b0fe72e4b6cd5a8515efe97a5b0f8a76edcac2/develop/dev-guide-choose-driver-or-orm.md?plain=1#L275-L278

Reply:
Done